market/views.py

```diff
@@ -64,25 +64,30 @@ def post(self, request, *args, **kwargs):
 
             if quantity > 0:
                 if mode == 'buy':
-                    purchase_amount = Decimal(quantity)*price
-                    if user.cash >= purchase_amount:
-                        if company.stocks_remaining >= quantity:
-                            obj = Transaction.objects.create(
-                                user=user,
-                                company=company,
-                                num_stocks=quantity,
-                                price=price,
-                                mode=mode,
-                                user_net_worth=InvestmentRecord.objects.calculate_net_worth(user)
-                            )
-
-                            messages.success(request, 'Transaction Complete!')
+                    # Checking with max stocks a user can purchase for a company
+                    total_quantity = investment_obj.stocks + quantity
+                    if not total_quantity > company.max_stocks_sell:
```

Comment:
Convert this to a simple comparison expression:  
`if total_quantity <= company.max_stocks_sell`  
This is more readable.

Reply:
The expression was changed as directed and is working fine